src/ClimaLand.jl

```diff
@@ -424,4 +424,5 @@ import .Diagnostics: default_diagnostics
 
 # Simulations
 include(joinpath("simulations", "Simulations.jl"))
+include("ext/FluxnetSimulationsExt.jl")
```

Comment:
This is not needed, right?

Reply:
Correct - we can delete this and it should fix the ClimaLand precompilation error that's in CI right now